### ISSUE TITLE
Add NIRS modality to UI

### ISSUE THREAD

**Issue by effigies** [OpenNeuroOrg/openneuro#2712]
NIRS support was added in BIDS 1.8.0 / bids-validator 1.9.8.

At a minimum we need to make things like https://openneuro.org/search/modality/nirs work. In the long run, it would be good to have UI elements like the dropdown and a new modality cube.

**Comment by effigies** [comment]
@rob-luke You wouldn't happen to have an image that would make a good cube, would you?

Current cubes:

![image](https://user-images.githubusercontent.com/83442/199817531-374a44e1-d790-4008-b1a7-ff546efcd4eb.png)

**Comment by rob-luke** [comment]
Thanks for kicking this off @effigies 
And yes, I'll find an appropriate image over the weekend. Great idea!!

**Comment by effigies** [comment]
Here's a tool for extending a color palette that might be useful for selecting a NIRS color: https://github.com/lmcinnes/glasbey

**Comment by nellh** [comment]
@rob-luke Just a ping to see if you came up with something, I'd like to add NIRS support to our web app.

**Comment by mayucel** [comment]
@effigies  and @nellh , Here is a proposal for the NIRS cube for the OpenNeuro website. Please let me know if it looks ok or you need to orig ppt!

![Image](https://github.com/user-attachments/assets/7ac6e46d-dabc-4859-9ce6-6b19765b2e24)

**Comment by nellh** [comment]
> [@effigies](https://github.com/effigies) and [@nellh](https://github.com/nellh) , Here is a proposal for the NIRS cube for the OpenNeuro website. Please let me know if it looks ok or you need to orig ppt!
> 
> ![Image](https://github.com/user-attachments/assets/7ac6e46d-dabc-4859-9ce6-6b19765b2e24)

Hi! Thanks for that. The cubes are rendered as part of the web app, so we'd just need a single image that can be cropped to fit the diamond shape.

**Comment by mayucel** [comment]
@nellh , do you mean the one I shared works for you as this is a single image? Could you clarify?

**Comment by nellh** [comment]
> [@nellh](https://github.com/nellh) , do you mean the one I shared works for you as this is a single image? Could you clarify?

No, we'd need your source image, instead of the composed version that's already displayed as the faces of the cube.

**Comment by mayucel** [comment]
Like these below?

![Image](https://github.com/user-attachments/assets/bf5e4185-86c9-43c8-993a-07d9bff23532)

![Image](https://github.com/user-attachments/assets/693ed849-b46e-48d7-8e60-7febfccfa8bc)

![Image](https://github.com/user-attachments/assets/4a79246b-7736-4030-b4b0-f37dac118241)

**Comment by effigies** [comment]
@mayucel Here are the existing images:

* https://github.com/OpenNeuroOrg/openneuro/blob/master/packages/openneuro-components/src/assets/ieeg.jpg
* https://github.com/OpenNeuroOrg/openneuro/blob/master/packages/openneuro-components/src/assets/eeg.jpg
* https://github.com/OpenNeuroOrg/openneuro/blob/master/packages/openneuro-components/src/assets/meg.jpg
* https://github.com/OpenNeuroOrg/openneuro/blob/master/packages/openneuro-components/src/assets/mri.jpg
* https://github.com/OpenNeuroOrg/openneuro/blob/master/packages/openneuro-components/src/assets/pet-scan.jpg

The faces are done with CSS, so we do not need it angled.

**Comment by mayucel** [comment]
@effigies , that makes a lot of sense! (Guess I did not have to figure out how to angle it right myself:) Do you also need the one for the front face which says NIRS? Here is the sides:

![Image](https://github.com/user-attachments/assets/145d4dd3-b49d-412d-9903-dab7836fbc3e)

**Comment by effigies** [comment]
That should be enough for us to go on, thanks! What is the source and license of this image?

**Comment by mayucel** [comment]
@effigies , the image is generated using OpenAI's DALL·E tool, prompted through ChatGPT (by me).

**Comment by effigies** [comment]
Hmm. I'm not sure if we can use that. I don't think there is definitive case law that AI-generated images are considered fair use of the training datasets. Although OpenAI does not assert copyright to images generated by DALL-E, that doesn't address the question of the original artists' copyright.

I wonder about something like the final images in https://www.fieldtriptoolbox.org/tutorial/nirs/nirs_multichannel/#plot-results? Is that something you'd be able to make from your data (or some that is available on OpenNeuro), which would have unambiguous copyright? Would a figure like that be recognizable for fNIRS researchers?

cc @robertoostenveld since FieldTrip evidently supports these data. I wonder if you have anything that we could use with credit.

**Comment by robertoostenveld** [comment]
Do you see anything visually attractive in https://pmc.ncbi.nlm.nih.gov/articles/PMC11369826/ or in https://www.sciencedirect.com/science/article/pii/S1053811923000290 ? Those are both CC licensed, so can be reused with appropriate credit. Or I can ask the primary author (Helena Cockx, @helenacockx) whether you can have them "for free". 

Perhaps a cropped section of figure 3 in the 2nd paper: 
![](https://ars.els-cdn.com/content/image/1-s2.0-S1053811923000290-gr3_lrg.jpg)

**Comment by mayucel** [comment]
@effigies and @robertoostenveld , I really loved what I came up with, the AI one, but if that is not possible, I will see what else we can have. Thanks for the recommendations but these look similar to EEG montage, I would prefer more like a brain image obtained via fNIRS data if it has to be an image based on real-data, or I will do something like the AI one myself, as the idea belongs to me...
